lib/src/api/engine/local/native.rs

```diff
@@ -146,6 +146,15 @@ pub(crate) fn router(
 			.with_query_timeout(address.config.query_timeout)
 			.with_transaction_timeout(address.config.transaction_timeout)
 			.with_capabilities(address.config.capabilities);
+		#[cfg(any(
+			feature = "kv-surrealkv",
+			feature = "kv-file",
+			feature = "kv-rocksdb",
+			feature = "kv-fdb",
+			feature = "kv-tikv",
+			feature = "kv-speedb"
+		))]
```

Comment:
The fact that we are using a local engine means that at least one of these is already enabled.